elastalert/elastalert.py

```diff
@@ -994,6 +1010,25 @@ def start(self):
             sleep_duration = total_seconds(next_run - datetime.datetime.utcnow())
             self.sleep_for(sleep_duration)
 
+    def wait_until_responsive(self, timeout=None, clock=timeit.default_timer):
+        """Wait until ElasticSearch becomes responsive (or too much time passes)."""
+
+        # Elapsed time is a floating point number of seconds.
+        timeout = timeout.total_seconds()
+
+        # Periodically poll ElasticSearch.  Keep going until ElasticSearch is
+        # responsive *and* the writeback index exists.
+        ref = clock()
+        while (timeout is None) or ((clock() - ref) < timeout):
+            try:
+                if self.writeback_es.indices.exists(self.writeback_index):
```

Comment:
Can we add

```
else:
    logging.error('ElastAlert status index "{}" does not exist. '
                           'Please run elastalert-create-index'.format(self.writeback_index))
    exit(1)
```

or something similar?

I don't want someone to get stuck in an infinite loop if they have failed to create the index or have a typo in their config.